### PR TITLE
chore: release notes for v1.49 js

### PR DESCRIPTION
**Pull request by dgozman**

### CODE REVIEW

File: docs/src/release-notes-js.md [hunk 1]
```diff
@@ -8,6 +8,48 @@ import LiteYouTube from '@site/src/components/LiteYouTube';
...
+
+### Miscellaneous
+
+- `<canvas>` elements inside a snapshot now draw a preview.
+- New method [`method: Tracing.group`] to visually group actions in the trace.
+- Playwright docker images switched from Node.js v20 to Node.js v22 LTS.
+
+### Breaking changes
+
+- There will be no more updates for WebKit on Ubuntu 20.04. We recommend updating Ubuntu to a later version.
```
Comment (mxschmitt) [inline docs/src/release-notes-js.md:49]: Lets mention Debian 11 here as well.

File: docs/src/release-notes-js.md [hunk 2]
```diff
@@ -8,6 +8,48 @@ import LiteYouTube from '@site/src/components/LiteYouTube';
 
 ## Version 1.49
 
+### Aria snapshots
+
+New assertion [`method: LocatorAssertions.toMatchAriaSnapshot`] provides a way to verify page structure by comparing it with an expected YAML representation of the accessibility tree.
```
Comment (Skn0tt) [inline docs/src/release-notes-js.md:13]: nit: making this a little more succinct.

```suggestion
The new assertion [`method: LocatorAssertions.toMatchAriaSnapshot`] verifies page structure by comparing to an expected accessibility tree, represented as YAML.
```

**Review by Skn0tt — APPROVED** [review]
looks good to me!

File: docs/src/release-notes-js.md [hunk 3]
```diff
@@ -8,6 +8,48 @@ import LiteYouTube from '@site/src/components/LiteYouTube';
...
+- New option [`property: TestConfig.tsconfig`] allows to specify a single `tsconfig` to be used for all tests.
+- New method [`method: Test.fail.only`] to focus on a failing test.
+- Options [`property: TestConfig.globalSetup`] and [`property: TestConfig.globalTeardown`] now support multiple setups/teardowns.
+- New value `'on-first-failure'` for [`property: TestOptions.screenshot`].
+- Added "previous" and "next" buttons to the HTML report to quickly switch between test cases.
+- New properties [`property: TestInfoError.cause`] and [`property: TestError.cause`] mirroring [`Error.cause`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause).
+
+### Miscellaneous
+
+- `<canvas>` elements inside a snapshot now draw a preview.
```
Comment (mxschmitt) [inline docs/src/release-notes-js.md:43]: Lets wait with landing until https://github.com/microsoft/playwright/issues/33562#issuecomment-2471874482 is resolved?

Comment (Skn0tt) [reply docs/src/release-notes-js.md:43]: Agree!

Comment (dgozman) [reply docs/src/release-notes-js.md:43]: We'll update if needed.

**Pull request merged**